docs(d1): :memo: add persist flag usage instruction for v3 (#8965)
cloudflare/cloudflare-docs: content/d1/get-started.md

While in your project directory, test your database locally by running:

```sh

> $ wrangler dev --local --persist

Reviewer: This is only the case for wrangler 3, which not everyone is using yet.

Author: I have added a new blockquote explaining it's only from Wrangler 3.

$ wrangler dev --local --persist
```

> > Starting from Wrangler 3 you don't need to pass the `--persist` flag. By default it'll be cached in the `.wrangler` folder and used upon reloads.

Reviewer 2: ```suggestion
> If using Wrangler 3, you do not need to pass the `--persist` flag. By default, it will be cached in the `.wrangler` folder and used on reloads.
```
